cmd/gh/main.go

```diff
@@ -92,6 +92,11 @@ func main() {
 		}
 	}
 
+	if cmd == nil {
+		fmt.Fprintf(stderr, "failed to parse command, %s\n", err)
+		os.Exit(2)
```

Comment:
So, the reason why I originally suggested just doing `cmd != nil && cmdutil.IsAuthCheckEnabled(cmd)` instead of explicitly handling errors is because `rootCmd.ExecuteC()` will be executed below this point and its error will be handled in a way that covers a wider variety of errors. I still think that should be the way to go, so for the sake of simplicity, please leave error handling to the below block, and just avoid a crash from happening when checking auth.

Reply:
ah I see, my apologies I misunderstood the original request. I have made the changes and pushed them. In the -R case the program even correctly fetches the issue :)
```
$ ./gh -w view repo cli/cli
unknown shorthand flag: 'w' in -w

Usage:  gh repo <command> [flags]

Available commands:
  clone
  create
  fork
  view

$ ./gh -R cli/cli issue view 1110
Markdown rendering show softbreaks
Open • KangOl opened about 3 months ago • 4 comments

Labels: bug, help-wanted


  ### Describe the bug                                                        
....                                                                                                  
```

Reply:
Interesting! So you can put `-R` as early as `gh -R owner/repo issue list` and it's just the same as putting it in the end; I did not know that!